package.json

```
@@ -89,6 +89,7 @@
   "devDependencies": {
     "@stylelint/prettier-config": "^3.0.0",
     "@stylelint/remark-preset": "^5.0.0",
+    "awesome-stylelint": "github:stylelint/awesome-stylelint#prepare-website",
```

```suggestion
    "awesome-stylelint": "github:stylelint/awesome-stylelint",
```

Then, update `package-lock.json`:

```shell
npm i
```